src/components/PostPageDetail/index.js

```diff
@@ -263,6 +249,46 @@ const PostPageDetailIdComponent = (props) => {
     }
   };
 
+  const commentChildren = async (isAnonimity, anonimityData) => {
+    setLoadingPost(true);
+    try {
+      if (textComment.trim() !== '') {
+        const data = await createChildCommentV3(
+          textComment,
+          reactionId,
+          true,
+          item.id,
+          isAnonimity,
+          anonimityData
+        );
+        scrollViewRef.current.scrollToEnd();
+        if (data.code === 200) {
+          setTextComment('');
+          setReplyUsername('');
+          setTypeComment('parent');
+          updateFeed(true);
+        } else {
+          Toast.show(StringConstant.generalCommentFailed, Toast.LONG);
+          setLoadingPost(false);
+        }
+      }
+    } catch (error) {
+      Toast.show(StringConstant.generalCommentFailed, Toast.LONG);
+      setLoadingPost(false);
+    }
+  };
+
+  const onComment = (isAnonimity, anonimityData) => {
+    if (typeComment === 'parent') {
+      commentParent(isAnonimity, anonimityData);
+      if (props?.refreshParent) {
+        props.refreshParent();
+      }
+    } else {
+      commentChildren(isAnonimity, anonimityData);
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new function `commentChildren` has been added, and the `onComment` function has been updated to call either `commentParent` or `commentChildren` based on the value of `typeComment`. This change allows for dynamic handling of parent and child comments, but ensure that all instances where `onComment` is used have been updated to handle this change.
```diff
+ const commentChildren = async (isAnonimity, anonimityData) => {
+   setLoadingPost(true);
+   try {
+     if (textComment.trim() !== '') {
+       const data = await createChildCommentV3(
+         textComment,
+         reactionId,
+         true,
+         item.id,
+         isAnonimity,
+         anonimityData
+       );
+       scrollViewRef.current.scrollToEnd();
+       if (data.code === 200) {
+         setTextComment('');
+         setReplyUsername('');
+         setTypeComment('parent');
+         updateFeed(true);
+       } else {
+         Toast.show(StringConstant.generalCommentFailed, Toast.LONG);
+         setLoadingPost(false);
+       }
+     }
+   } catch (error) {
+     Toast.show(StringConstant.generalCommentFailed, Toast.LONG);
+     setLoadingPost(false);
+   }
+ };
+ 
+ const onComment = (isAnonimity, anonimityData) => {
+   if (typeComment === 'parent') {
+     commentParent(isAnonimity, anonimityData);
+     if (props?.refreshParent) {
+       props.refreshParent();
+     }
+   } else {
+     commentChildren(isAnonimity, anonimityData);
+   }
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->